app/admin/dashboard.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+ActiveAdmin.register_page 'Dashboard' do
```

Comment:
remove this page

Reply:
In next steps i use this page for SideKiq button that brings admin to /sidekiq page. I dont know if i doing right by it was not really comfortable when i was working with sidekiq to check if my task are running well so i use dashboard for this purposes.